wagtailvideos/views/videos.py

```diff
@@ -76,7 +75,7 @@ def index(request):
 @permission_checker.require('change')
 def edit(request, video_id):
     VideoForm = get_video_form(Video)
-
+    VideoForm.use_required_attribute = False
```

Comment:
It'll still raise a validation error if the title field is not filled out, it just won't stop it at a browser level